### PR TITLE
NI-DCPower: Add LCR source AC voltage example

### PR DESCRIPTION
**Pull request by olsl21**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/nimi-python/blob/master/CONTRIBUTING.md).
- [x] I've updated [CHANGELOG.md](https://github.com/ni/nimi-python/blob/master/CHANGELOG.md) if applicable.
- [x] I've added tests applicable for this pull request

### What does this Pull Request accomplish?

- Add "nidcpower_lcr_source_ac_voltage.py" example to demonstrate how to perform a simple AC voltage sourcing and take LCR measurements.

### What testing has been done?

- Unit tests added as part of the example file passed.
- Ran the example manually, the output was as follows:
```
> python nidcpower_lcr_source_ac_voltage.py -n 4190 -c NI_STANDARD_1M
Channel             : 4190/0
DC voltage          : 4.34928e-07 V
DC current          : 1.77324e-08 A
Stimulus frequency  : 10,000 Hz
AC voltage          : 0.7+0j V RMS
AC current          : 7.12116e-05+1.68737e-07j A RMS
Impedance           : 9,829.81-23.292j Ω
Impedance magnitude : 9,829.84 Ω
Impedance phase     : -0.135763 °
Admittance          : 0.000101731+2.41053e-07j S
Admittance magnitude: 0.000101731 S
Admittance phase    : 0.135763 °
Series inductance   : -0.000370703 H
Series capacitance  : 6.83304e-07 F
Series resistance   : 9,829.81 Ω
Parallel inductance : -66.0248 H
Parallel capacitance: 3.83648e-12 F
Parallel resistance : 9,829.86 Ω
Dissipation factor  : 422.026
Quality factor      : 0.00236952
Measurement mode    : LCR
DC in compliance    : False
AC in compliance    : False
Unbalanced          : False
```

### CODE REVIEW

File: src/nidcpower/examples/nidcpower_lcr_source_ac_voltage.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+        session.lcr_voltage_amplitude = lcr_voltage_rms
+        session.lcr_dc_bias_source = lcr_dc_bias_source
+        session.lcr_dc_bias_voltage_level = lcr_dc_bias_voltage_level
+        session.lcr_measurement_time = lcr_measurement_time
+        session.lcr_custom_measurement_time = lcr_custom_measurement_time
+        session.lcr_source_delay_mode = lcr_source_delay_mode
+        session.source_delay = source_delay
+
+        with session.initiate():
+            session.wait_for_event(nidcpower.Event.SOURCE_COMPLETE, 21.0)
```
Comment (sbethur) [inline src/nidcpower/examples/nidcpower_lcr_source_ac_voltage.py:38]: Specify these as keyword args; it improves readability which is essential in examples.

File: src/nidcpower/examples/nidcpower_lcr_source_ac_voltage.py [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+        args.lcr_frequency,
+        args.lcr_impedance_range,
+        getattr(nidcpower.CableLength, args.cable_length),
+        args.lcr_voltage_rms,
+        getattr(nidcpower.LCRDCBiasSource, args.lcr_dc_bias_source),
+        args.lcr_dc_bias_voltage_level,
+        getattr(nidcpower.LCRMeasurementTime, args.lcr_measurement_time),
+        args.lcr_custom_measurement_time,
+        getattr(nidcpower.LCRSourceDelayMode, args.lcr_source_delay_mode),
+        args.source_delay,
```
Comment (sbethur) [inline src/nidcpower/examples/nidcpower_lcr_source_ac_voltage.py:76]: Specify these as keyword args; it improves readability which is essential in examples.

File: src/nidcpower/examples/nidcpower_lcr_source_ac_voltage.py [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+        10.0e3,
+        100.0,
+        nidcpower.CableLength.NI_STANDARD_2M,
+        700.0e-3,
+        nidcpower.LCRDCBiasSource.OFF,
+        0.0,
+        nidcpower.LCRMeasurementTime.MEDIUM,
+        10.0e-3,
+        nidcpower.LCRSourceDelayMode.AUTOMATIC,
+        16.66e-3,
```
Comment (sbethur) [inline src/nidcpower/examples/nidcpower_lcr_source_ac_voltage.py:98]: Specify these as keyword args; it improves readability which is essential in examples.

**Review by sbethur — CHANGES_REQUESTED** [review]

File: src/nidcpower/examples/nidcpower_lcr_source_ac_voltage.py (hunk 3 above)
Comment (olsl21) [reply src/nidcpower/examples/nidcpower_lcr_source_ac_voltage.py]: I have updated it in my latest commit.

File: src/nidcpower/examples/nidcpower_lcr_source_ac_voltage.py [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+        session.lcr_voltage_amplitude = lcr_voltage_rms
+        session.lcr_dc_bias_source = lcr_dc_bias_source
+        session.lcr_dc_bias_voltage_level = lcr_dc_bias_voltage_level
+        session.lcr_measurement_time = lcr_measurement_time
+        session.lcr_custom_measurement_time = lcr_custom_measurement_time
+        session.lcr_source_delay_mode = lcr_source_delay_mode
+        session.source_delay = source_delay
+
+        with session.initiate():
+            session.wait_for_event(event_id=nidcpower.Event.SOURCE_COMPLETE, timeout=21.0)
```
Comment (sbethur) [inline src/nidcpower/examples/nidcpower_lcr_source_ac_voltage.py:38]: If there's a specific reason why 21 seconds is needed, a comment explaining why would be very useful. If no specific reason, I'd suggest using the default timeout value.

Comment (olsl21) [reply src/nidcpower/examples/nidcpower_lcr_source_ac_voltage.py]: Thanks @singerbistro for the explanation. I have added a comment and updated the timeout value to 5.0s in my latest commit:
```
# Low frequencies require longer settling times than the default timeout for
# wait_for_event(), hence 5.0s is set here as a reasonable timeout value
```

**Review by marcoskirsch — APPROVED** [review]

**Pull request merged**